location/gql_mutations.py

```diff
@@ -290,6 +293,11 @@ class Input(HealthFacilityInputType):
     @classmethod
     def async_mutate(cls, user, **data):
         try:
+            current_HF = HealthFacility.objects.get(uuid=data['uuid'])
```

Comment:
.get raises exception if object is not found, can you confirm that exception for invalid uuid in output is informative enough? If not please handle this scenario.

Reply:
I can't think of a proper scenario to generate that error. This code is run only when we are editing HF, so it must exist in the database. One thing that could happen is that someone deletes this instance while someone else is editing it, should I check that?

Reply:
No, that's fine - if user deletes entity it's still in the database, we can keep `get` in here